fox_consumer/fox_helper.py

```diff
@@ -5,6 +5,20 @@ class FoxItem(object):
     """
     The FoxItem class
     """
+    schema = {
```

Comment:
its just json standard - I actually though I deleted it before.
I am deleting it now...